catroid/src/main/java/org/catrobat/catroid/common/defaultprojectcreators/DefaultProjectCreatorDefault.java

```diff
@@ -163,11 +163,14 @@ public Project createDefaultProject(String projectName, Context context, boolean
 
 			ProjectManager.getInstance().getFileChecksumContainer().addChecksum(soundInfo1.getChecksum(), soundInfo1.getAbsolutePath());
 			ProjectManager.getInstance().getFileChecksumContainer().addChecksum(soundInfo2.getChecksum(), soundInfo2.getAbsolutePath());
+			ProjectManager.getInstance().getFileChecksumContainer().addChecksum(birdWingUpLookData.getChecksum(), birdWingUpLookData.getAbsolutePath());
+			ProjectManager.getInstance().getFileChecksumContainer().addChecksum(birdWingDownLookData.getChecksum(), birdWingDownLookData.getAbsolutePath());
+			ProjectManager.getInstance().getFileChecksumContainer().addChecksum(cloudLookData.getChecksum(), cloudLookData.getAbsolutePath());
```

Comment:
I think you should add the checksum for cloudLookData two times. The first time to create the checksum key and the second time to increment the usage (for the cloned look below).